elixirkit/demo/README.md

```diff
@@ -0,0 +1,11 @@
+# Demo
+
+This is an example ElixirKit application.
+
+See:
+
+  * [Elixir Server](lib/demo.ex)
+  * [Swift Client](rel/swift)
+  * [AppKit Client](rel/appkit)
```

Comment:
Curiosity: why do we need both Swift and AppKit clients?

Reply:
Ah, the Swift one is just Swift, no UI toolkit?

Reply:
Yup, no UI!

for similiar reason we have dotnet without UI which we can actually use on Mac which is really convenient.

Reply:
We should be able to use swift no UI on Windows too!